src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -116,7 +116,7 @@
   <ItemGroup>
     <PackageReference Include="DnsClient" Version="1.3.1" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" PrivateAssets="All" />
-    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.0.0" />
+    <PackageReference Include="MongoDB.Libmongocrypt" Version="0.0.0-local" />
```

Comment:
I assume you will fix this before pushing to master.

Reply:
Correct